.gitignore

```diff
@@ -404,3 +404,9 @@ python/build/
 /go/okapi/coverage.out
 /go/okapi/test_output.xml
 ruby/Gemfile.lock
+libokapi.a
+ios/Pods
+ios/okapi.xcodeproj/project.xcworkspace/xcuserdata
```

Comment:
Mind using the GitHub default ignore template for the subfolder?
https://github.com/github/gitignore/blob/master/Global/Xcode.gitignore

Reply:
Not at all, let me check the file first.

Reply:
Done!